app/models/mobile_user.rb

```diff
@@ -0,0 +1,9 @@
+class MobileUser < ApplicationRecord
+  has_many :mobile_comments, foreign_key: :id
+  belongs_to :auth_token
+
+  def image_url
+    "https://api.adorable.io/avatars/80/#{auth_token.value}.png"
```

Comment:
Rozumiem, że to się zmieni w przyszłości na generowanie losowe. Mimo wszystko adres url powinien być przeniesiony do opcji konfiguracyjnych, albo stałej w jakimś serwisie, który będzie odpowiadał za generowanie avatarów, np. `AvatarGenerator`.